### ISSUE TITLE
`call_agenda()` error in system.file()

### ISSUE THREAD

**Issue by jules32** [Openscapes/kyber#33]
This is something that @stefaniebutland and I can likely figure out with screensharing since she was able to create Call 5 Agenda with no error, but thought I'd document it here: 

I'm able to run `call_agenda()` no problem with `call_number` = 2, 3, 4,  but it errors with 1, 5. The error: 

```r

kyber::call_agenda(registry_url = "https://docs.google.com/spreadsheets/d/1Ys9KiTXXmZ_laBoCV2QWEm7AcnGSVQaXvm2xpi4XTSc/edit#gid=695033382",cohort_id = "2022-noaa-sefsc-summer", call_number = 5)
✔ Reading from OpenscapesChampionsCohortRegistry.
✔ Range ''cohort_metadata''.
New names:
• `` -> `...13`
• `` -> `...14`
• `` -> `...15`
• `` -> `...16`
• `` -> `...17`
• `` -> `...18`
• `` -> `...19`
• `` -> `...20`
✔ Reading from OpenscapesChampionsCohortRegistry.
✔ Range ''call_metadata''.
Error in `system.file()`:
! Paths can't start with `inst`
ℹ Files in `inst` are installed at top-level.
Run `rlang::last_error()` to see where the error occurred.

## and then:
rlang::last_error()
<error/rlang_error>
Error in `system.file()`:
! Paths can't start with `inst`
ℹ Files in `inst` are installed at top-level.
---
Backtrace:
 1. kyber::call_agenda(...)
 3. pkgload (local) system.file("inst", "agendas", files, package = "kyber")
Run `rlang::last_trace()` to see the full context.

## and then:
rlang::last_trace()
<error/rlang_error>
Error in `system.file()`:
! Paths can't start with `inst`
ℹ Files in `inst` are installed at top-level.
---
Backtrace:
    ▆
 1. └─kyber::call_agenda(...)
 2.   ├─base::file.copy(...) at kyber/R/call_agenda.R:103:4
 3.   └─pkgload (local) system.file("inst", "agendas", files, package = "kyber")
 4.     └─cli::cli_abort(c("Paths can't start with `inst`", i = "Files in `inst` are installed at top-level."))
 5.       └─rlang::abort(message, ..., call = call, use_cli_format = TRUE)

**Comment by seankross** [comment]
This is an implementation error that is my fault! Fixed in #35. Please reinstall and test it again.

**Comment by jules32** [comment]
You are so awesome @seankross , thank you! Tested with Agendas 1-5 (for good measure) and all is working perfectly. Thank you!
